### PR TITLE
Fixed syntax error in sh script header

### PR DESCRIPTION
**Pull request by olehermanse**
Introduced in d932325141c9353fe7691d7cefab8950bea10adf.

### CODE REVIEW

**Comment by olehermanse** [comment]
@cf-bottom jenkins with exotics

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2680/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/2680/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-2680/

File: packaging/common/script-templates/script-common-header-last.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ is_nova()
   test "$PROJECT_TYPE" = "cfengine-nova" || test "$PROJECT_TYPE" = "cfengine-nova-hub"
 }
 
-case os_type() in
+case "`os_type`" in
```
Comment (nickanderson) [inline packaging/common/script-templates/script-common-header-last.sh]: :( i suck

Comment (olehermanse) [reply packaging/common/script-templates/script-common-header-last.sh]: ~:( i suck~ sh sucks

**Pull request merged**